CHANGELOG.latest.md

```diff
@@ -1,33 +1,28 @@
+[Full Changelog](https://github.com/revenuecat/purchases-ios/compare/4.0.0-beta.rc.1...4.0.0-rc.2)
 - See our [RevenueCat V4 API update doc](docs/V4_API_Updates.md) for API updates.
 
-RC 1 introduces the following updates:
+RC 2 introduces the following updates:
 
 ### API changes:
 
-- `Purchases.paymentDiscount(forProductDiscount:product:completion:)` and `Purchases.paymentDiscount(forProductDiscount:product:)` have been removed. Now, instead of obtaining the `SKPaymentDiscount` from a `SKProductDiscount` to then call `purchase(package:discount:)`, you check eligibility for the promo offer by calling `checkPromotionalDiscountEligibility(forProductDiscount:product:)`, then get the `StoreProductDiscount` directly from the `StoreProduct` and pass that into `purchase(package:discount:)`. 
+- Removed `SubscriptionPeriod.Unit.unknown`. Subscriptions with empty `SubscriptionPeriod` values will have `nil` `subscriptionPeriod` instead.
+- Removed `StoreProductDiscount.none`, since it wasn't needed.
 
-- `StoreProduct` and `StoreProductDiscount`, replace `SKProduct` and `SKProductDiscount` in the following methods:
-    - `Purchases.getProducts(_:completion:)`
-    - `Purchases.products(_:)`
-    - `Purchases.purchase(product:completion:)`
-    - `Purchases.purchase(product:)`
-    - `Purchases.purchase(product:discount:completion:)`
-    - `Purchases.purchase(product:discount:)`
-    - `Purchases.purchase(package:discount:completion:)`
-    - `Purchases.purchase(package:discount:)`
-    - `PurchasesDelegate.purchases(shouldPurchasePromoProduct:defermentBlock:)`
-- `StoreProduct.introductoryPrice` has been renamed to `StoreProduct.introductoryDiscount`
-- `StoreTransaction` now includes `quantity`
-- Renamed `Purchases.restoreTransactions` to `Purchases.restorePurchases`
-- Lowered `StoreProduct.introductoryDiscount` availability to iOS 11.2 and equivalent OS versions
-- Added several `@available` annotations for automatic migration from StoreKit types
+### Documentation: 
 
-In addition to all of the changes from beta 10, [listed here.](https://github.com/RevenueCat/purchases-ios/blob/4.0.0-beta.10/CHANGELOG.latest.md)
+- Documentation is now using DocC and it can be found in https://revenuecat-docs.netlify.app/documentation/Revenuecat. 
+- We've made several improvements to docstrings and added a few landing pages for the most important sections of the SDK. 
 
+### Migration fixes
+
+- Fixed a few instances where Xcode's automatic migration tools wouldn't correctly update the code.
 
 ### Other changes: 
 - There are lots of under the hood improvements. If you see any issues we'd appreciate [bug reports](https://github.com/RevenueCat/purchases-ios/issues/new?assignees=&labels=bug&template=bug_report.md&title=)!
+
+### Changes from previous RC
+
+These changes add to all of the changes from beta RC 1, [listed here.](https://github.com/RevenueCat/purchases-ios/blob/4.0.0-rc.1/CHANGELOG.latest.md).
```

Comment:
all auto-generated from this point on, other than the podspec and fastfile at the very bottom of the PR